txpipe/photoz_stack.py

```diff
@@ -317,7 +317,7 @@ def run(self):
         # Data we need - the photometry catalog for DC2 has the true redshift
         # value in, and the tomography catalog has the binning.
         photo_iterator = self.iterate_hdf(
-            'photometry_catalog', # tag of input file to iterate through
+            'shear_photometry_catalog', # tag of input file to iterate through
```

Comment:
I missed this, so we are separating the photometry catalogs? also we should probably unify what we want to use sources or shear :)

Reply:
mistake - fixed!